internal/db/rdb/mysql/mysql.go

```diff
@@ -168,8 +178,17 @@ func (m *mySQLClient) Ping(ctx context.Context) (err error) {
 }
 
 // Close closes the connection of MySQL database.
-// If the connection is already closed or closing conncection is failed, it returns error.
+// If the connection is already closed or closing connection is failed, it returns error.
 func (m *mySQLClient) Close(ctx context.Context) (err error) {
+	if m.session == nil {
+		err = errors.ErrMySQLSessionNil
+		log.Debugf(
+			"err: %s, { db: %s, host: %s, port: %d, user: %s, pass: %s, name: %s, charset: %s } ",
+			err.Error(), m.db, m.host, m.port, m.user, m.pass, m.name, m.charset,
+		)
+		return err
+	}
+
```

Comment:
I think we should not return error when session is nil, do nothing instead.
```suggestion
```

Reply:
@kevindiu why not?
if the session is nil, it would be occurred panic

Reply:
but it is `Close()` function. If the session is nil I think it is better to return.
What do you think?

Reply:
@kpango @vankichi what do you think?